hack/release.sh

```diff
@@ -31,11 +37,24 @@ update_chart_yamls() {
     sed -i "s/\b${oldVersion}\b/value: ${version}/g" deploy-all-in-one-tls.yaml
 }
 
+publish_release() {
+    local version=$1
+
+    # create gh release
+    gothub release \
+	   --user infracloudio \
+	   --repo botkube \
+	   --tag $version \
+	   --name "$version" \
+	   --description "$version"
```

Comment:
Description will be version?  This will be changed later once changelog generated?

Reply:
We'll be manually changing that as changelog link needs to point on correct date

Reply:
LGTM